dependencyManagement/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ val errorProneVersion = "2.27.1"
 val prometheusVersion = "0.16.0"
 val mockitoVersion = "4.11.0"
 val slf4jVersion = "2.0.13"
+val contribIncubatingVersion = "1.25.0-alpha"
```

Comment:
could you rename this to `semConvVersion`

Reply:
fixed in 67fc874dd6297e2851edb906a6f274764a216606

Reply:
thanks